tests/queries/0_stateless/02968_mysql_show_warnings.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest
+# Tag no-fasttest: requires mysql client
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+${MYSQL_CLIENT} --execute "SHOW WARNINGS;"
+${MYSQL_CLIENT} --execute "show warnings;"
+${MYSQL_CLIENT} --execute "SHOW WARNINGS LIMIT 100;"
+${MYSQL_CLIENT} --execute "show warnings limit 100;"
+${MYSQL_CLIENT} --execute "SHOW WARNINGS LIMIT 100 OFFSET 100;"
```

Comment:
Looking at MySQL [docs](https://dev.mysql.com/doc/refman/8.0/en/show-warnings.html): 

```
SHOW WARNINGS [LIMIT [offset,] row_count]
```

`SHOW WARNINGS LIMIT 100, 50` (MySQL) is equivalent to `SHOW WARNINGS LIMIT 100 OFFSET 50` (ClickHouse). Did you ever see syntax `[offset,] row_count` in logs?

Reply:
I only saw plain `SHOW WARNINGS` so far.